common/src/main/java/io/netty/util/concurrent/AbstractEventExecutor.java

```diff
@@ -36,6 +36,8 @@
 
     private final EventExecutorGroup parent;
     private final Collection<AbstractEventExecutor> selfCollection = Collections.singleton(this);
+    
+    private RejectTaskHandler rejectTaskHandler = new DefaultRejectTaskHandler();
```

Comment:
It also needs to be renamed to `RejectedTaskHandler`, `rejectedTaskHandler`, and `DefaultRejectedTaskHandler`.